src/components/Input/DropDown/DropDownGroup.js

```diff
@@ -97,7 +106,30 @@ class DropDownGroup extends React.Component {
     return selectedItem && selectedItem.props.children;
   };
 
-  closeDropdown = () => this.setState({ isOpen: false });
+  cleanWord = throttle(() => {
```

Comment:
Does it make sense to clear throttle timeout in case if user typed smth and then started to use arrows to navigate?

Reply:
Again, there is no visible search field. The user does not see what he's typing. Ther eis no usecase to use the arrow keys to move the cursor.